components/hidden_data_box/wikis/apexlegends/hidden_data_box_custom.lua

```diff
@@ -0,0 +1,37 @@
+---
+-- @Liquipedia
+-- wiki=apexlegends
+-- page=Module:HiddenDataBox/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Variables = require('Module:Variables')
+
+local BasicHiddenDataBox = Lua.import('Module:HiddenDataBox', {requireDevIfEnabled = true})
+local CustomHiddenDataBox = {}
+
+function CustomHiddenDataBox.run(args)
+	BasicHiddenDataBox.addCustomVariables = CustomHiddenDataBox.addCustomVariables
+	return BasicHiddenDataBox.run(args)
+end
+
+function CustomHiddenDataBox.addCustomVariables(args, queryResult)
+	queryResult.extradata = queryResult.extradata or {}
+
+	Variables.varDefine('tournament_date', Variables.varDefault('tournament_enddate'))
+	Variables.varDefine('tournament_sdate', Variables.varDefault('tournament_startdate'))
+	Variables.varDefine('tournament_edate', Variables.varDefault('tournament_enddate'))
+	Variables.varDefine('date', Variables.varDefault('tournament_enddate'))
+	Variables.varDefine('sdate', Variables.varDefault('tournament_startdate'))
+	Variables.varDefine('edate', Variables.varDefault('tournament_enddate'))
+
+	Variables.varDefine('tournament_tier', Variables.varDefault('tournament_liquipediatier'))
+	Variables.varDefine('tournament_ticker_name', Variables.varDefault('tournament_tickername'))
+
+	Variables.varDefine('tournament_mode', Variables.varDefault('tournament_mode', 'team'))
```

Comment:
So there has recently been an introduction of a couple of different format tournaments that are not BR. 
But looking at how the infobox handles this at the moment, it could probably do with a rewrite in the infobox league end anyway, so I think maybe kick it for now